MdePkg/Test/Mock/Include/GoogleTest/Protocol/MockFirmwareVolume2.h

```diff
@@ -0,0 +1,148 @@
+     OUT       EFI_FV_ATTRIBUTES             *FvAttributes
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    SetVolumeAttributes,
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN OUT    EFI_FV_ATTRIBUTES             *FvAttributes
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    FvReadFile,
+
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN CONST  EFI_GUID                      *NameGuid,
+     IN OUT    VOID                          **Buffer,
+     IN OUT    UINTN                         *BufferSize,
+     OUT       EFI_FV_FILETYPE               *FoundType,
+     OUT       EFI_FV_FILE_ATTRIBUTES        *FileAttributes,
+     OUT       UINT32                        *AuthenticationStatus
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    ReadSection,
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN CONST  EFI_GUID                      *NameGuid,
+     IN        EFI_SECTION_TYPE              SectionType,
+     IN        UINTN                         SectionInstance,
+     IN OUT    VOID                          **Buffer,
+     IN OUT    UINTN                         *BufferSize,
+     OUT       UINT32                        *AuthenticationStatus
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    FvWriteFile,
+
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN        UINT32                        NumberOfFiles,
+     IN        EFI_FV_WRITE_POLICY           WritePolicy,
+     IN        EFI_FV_WRITE_FILE_DATA        *FileData
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    GetNextFile,
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN OUT    VOID                          *Key,
+     IN OUT    EFI_FV_FILETYPE               *FileType,
+     OUT       EFI_GUID                      *NameGuid,
+     OUT       EFI_FV_FILE_ATTRIBUTES        *Attributes,
+     OUT       UINTN                         *Size
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    GetInfo,
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN CONST  EFI_GUID                      *InformationType,
+     IN OUT    UINTN                         *BufferSize,
+     OUT       VOID                          *Buffer
+    )
+    );
+
+  MOCK_FUNCTION_DECLARATION (
+    EFI_STATUS,
+    SetInfo,
+    (
+     IN CONST  EFI_FIRMWARE_VOLUME2_PROTOCOL *This,
+     IN CONST  EFI_GUID                      *InformationType,
+     IN        UINTN                         BufferSize,
+     IN CONST  VOID                          *Buffer
+    )
+    );
+};
+
+MOCK_INTERFACE_DEFINITION (MockFirmwareVolume2Protocol);
+MOCK_FUNCTION_DEFINITION (MockFirmwareVolume2Protocol, GetVolumeAttributes, 2, EFIAPI);
+MOCK_FUNCTION_DEFINITION (MockFirmwareVolume2Protocol, SetVolumeAttributes, 2, EFIAPI);
+MOCK_FUNCTION_DEFINITION (MockFirmwareVolume2Protocol, FV_ReadFile, 7, EFIAPI);

```

Comment:
@VivianNK, did you mean to leave these with the prior naming convention?

Reply:
No, missed those. And newlines from last commit need to be removed.